src/components/Configs/ConfigList/ConfigListColumn.tsx

```diff
@@ -90,6 +91,13 @@ export const configListColumns: ColumnDef<ConfigItem, any>[] = [
       return <Status status={health} statusText={status} />;
     }
   },
+  {
```

Comment:
I think changes should go come before tags - Can we not use  min, size and maxSizes ? i.e. don't let name / type or status too short and let the other columns shrink to shrink